docs/UserGuide.md

```diff
@@ -197,10 +199,24 @@ To add a person to the contact list:
 `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [pr/PRICE] [i/IMPORTANT_INFORMATION] [s/STATUS] [t/TAG]…​`
 2. Press Enter.
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note on parameters used:**<br>
+* `PRICE` is used to indicate the price you have to pay for a person who you are hiring to help with your wedding, e.g. a
+  chef. You may choose to not specifying it if the contact is your family/relative/friend.
+* `IMPORTANT-INFORMATION` is used to indicate the important information that you need to take note of, e.g., if a
+  contact is practicing vegetarian diet or is only contactable through Whatsapp.
+* `STATUS` is used to indicate if a person is confirmed to attend your wedding. It can also be used to indicate if you
+  have confirmed to hire a person to help with your wedding.
+</div>
+
 <div markdown="block" class="alert alert-success">
 
 **:bulb: Tip:**<br>
 * A person can have any number of tags (including 0)
+* Since a wedding can be big and two or more persons might share contact methods with his/her family member(s), a person 
+  can have a phone number and/or email that is existing in the contact list (duplicate phone number and email is 
```

Comment:
Maybe you can specify that all contact information(other than the contact's name) can be duplicated.